docs/variables.md

```diff
@@ -4,108 +4,106 @@ linkTitle: "Variable Substitutions"
+of variables. This section lists the variables exposed by each type. You can access a
+variable via `resources.inputs.<resourceName>.<variableName>` or
+`resources.outputs.<resourceName>.<variableName>`.
 
-Each variable is accessible via `resources.inputs.<resource name>.<variable name>` or
-`resources.outputs.<resource name>.<variable name>`.
-
-#### Git PipelineResource
+#### Variables for the `Git` type
 
 | Variable | Description |
 | -------- | ----------- |
-| `name` | The resource's name |
-| `type` | `"git"` |
-| `url` | The URL to the Git repo |
-| `revision` | The revision to be checked out. |
-| `depth` | The integer value of the resource's `depth` param. |
-| `sslVerify` | The value of the resource's `sslVerify` param: `"true"` or `"false"`. |
-| `httpProxy` | The value of the resource's `httpProxy` param. |
-| `httpsProxy` | The value of the resource's `httpsProxy` param. |
-| `noProxy` | The value of the resource's `noProxy` param. |
-
-#### PullRequest PipelineResource
+| `name` | The name of the resource. |
+| `type` | Type value of `"git"`. |
+| `url` | The URL of the Git repository. |
+| `revision` | The revision to check out. |
+| `depth` | The integer value of the resource's `depth` parameter. |
+| `sslVerify` | The value of the resource's `sslVerify` parameter, either `"true"` or `"false"`. |
+| `httpProxy` | The value of the resource's `httpProxy` parameter. |
+| `httpsProxy` | The value of the resource's `httpsProxy` parameter. |
+| `noProxy` | The value of the resource's `noProxy` parameter. |
+
+#### Variables for the `PullRequest` type
 
 | Variable | Description |
 | -------- | ----------- |
-| `name` | The resource's name |
-| `type` | `"pullRequest"` |
-| `url` | The URL pointing to the pull request. |
-| `provider` | `"github"` or `"gitlab"`. |
-| `insecure-skip-tls-verify` | The value of the resource's `insecure-skip-tls-verify` param: `"true"` or `"false"`. |
+| `name` | The name of the resource. |
+| `type` | Type value of `"pullRequest"`.|
+| `url` | The URL of the pull request. |
+| `provider` | Provider value, either `"github"` or `"gitlab"`. |
+| `insecure-skip-tls-verify` | The value of the resource's `insecure-skip-tls-verify` parameter, either `"true"` or `"false"`. |
 
-#### Image PipelineResource
+#### Variables for the `Image` type
 
 | Variable | Description |
 | -------- | ----------- |
-| `name` | The resource's name |
-| `type` | `"image"` |
+| `name` | The name of the resource. |
+| `type` | Type value of `"image"`. |
 | `url` | The complete path to the image. |
-| `digest` | The image's digest. |
+| `digest` | The digest for the image. |
 
-#### GCS PipelineResource
+#### Variables for the `GCS` type
 
 | Variable | Description |
 | -------- | ----------- |
-| `name` | The resource's name |
-| `type` | `"gcs"` |
-| `location` | The location of the blob storage. |
+| `name` | The name of the resource. |
+| `type` | Type value of `"gcs"`. |
+| `location` | The path to the blob storage. |
```

Comment:
V minor again: technically it'll be a fully-qualified address like `gs://my-bucket/foo/bar`

Reply:
Reworded. :)